src/types/keyframes.ts

```diff
@@ -0,0 +1,110 @@
+import { BoxCssProps, CssProps } from './enhancers'
+
+export type KeyframesPercentageKey =
```

Comment:
I wonder if we want to consider something like this?
```
type Enumerate<N extends number, Acc extends number[] = []> = Acc['length'] extends N
  ? Acc[number]
  : Enumerate<N, [...Acc, Acc['length']]>

type IntRange<F extends number, T extends number> = Exclude<Enumerate<T>, Enumerate<F>>

type T = IntRange<20, 300>
```
requires TS 4.5 though (from SO)

Reply:
I know 😢  That was the first thing I looked into bc I didn't want to type out 0-100 manually 😂 (ended up running `new Array(100).fill(null).map((_, i) => i)` in a repl to enumerate all of the values) We'll have to give this repo some love and upgrade TS next!